iAM-legacy/Simulation/Investments.cs

```diff
@@ -148,11 +152,16 @@ public void LoadBudgets()
             String strYear;
 
             String strAmount;
-
+            MaximumYear = Int32.MinValue;
```

Comment:
`MaximumYear` is only used in the investment class. Make it private.

Reply:
Made private